Document the way to file pull requests (#558)
fedora-infra/the-new-hotness: docs/user-guide.rst

`Anitya <https://release-monitoring.org>`_. For the information how to create
a new project, look at the
`user guide in Anitya documentation <https://anitya.readthedocs.io/en/stable/user-guide.html>`_.

> Filling pull requests on dist-git

Reviewer: The right word is `filing`, but I think `opening` is a better fit here.

```suggestion
Opening pull requests in dist-git
```

---------------------------------

Thanks to the work of `Packit <https://packit.dev/>`_ Team it is now possible to have

> pull requests filled directly in `dist git <https://src.fedoraproject.org>`_ when new

Reviewer: ```suggestion
pull requests filed directly in `dist git <https://src.fedoraproject.org>`_ when a new
```

> update is detected by The New Hotness for monitored project.

Reviewer: ```suggestion
update is detected by The New Hotness for a monitored project.
```

> The process is described in the

Reviewer: ```suggestion
The process is described in
```
